dashboard/app/assets/stylesheets/application.css.scss

```diff
@@ -1936,6 +1947,16 @@ $modal-no-icon-gap: 22px;
   min-width: 100px;
 }
 
+#hint-button {
+  background-color: $lightest_gray;
+  border: 1px solid $light_gray;
+  color: black;
+
+  svg {
+    margin: -12px -6px;
+  }
+}
+
```

Comment:
They technically could right now, but the next item on my list is to restyle the "show hint" button for feedback blocks to match this, and I'll want the styles to live here when I do that.